tests/unit/data/execute_query/test_checksum.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from unittest import mock
+import warnings
+
+with warnings.catch_warnings(record=True) as suppressed_warning:
+    warnings.warn("Supressed warning", RuntimeWarning)
+
+
+def test_import_warning_is_rewritten():
+    with mock.patch(
+        "google.cloud.bigtable.data.execute_query._checksum.import_warning",
+        suppressed_warning,
+    ):
+        with warnings.catch_warnings(record=True) as import_warning:
+            from google.cloud.bigtable.data.execute_query._checksum import _CRC32C
+
+            assert import_warning == []
+            with warnings.catch_warnings(record=True) as first_call_warning:
+                assert _CRC32C.checksum(b"test") == 2258662080
+                assert (
+                    "Using pure python implementation of `google-crc32` for ExecuteQuery response validation"
+                    in str(first_call_warning[0])
+                )
```

Comment:
nit: it would be good to make sure repeated calls only send one warning. And maybe also a test where no warnings are needed

Reply:
Done